wikipediapreview.php

```diff
@@ -95,8 +95,20 @@ function myguten_set_script_translations() {
 	wp_set_script_translations( 'wikipedia-preview-localization', 'wikipedia-preview' );
 }
 
+function register_detectlinks_postmeta() {
+	$options = array(
+		'show_in_rest'  => true,
+		'auth_callback' => true,
+		'single'        => true,
+		'type'          => 'boolean',
+		'default'       => true, // it could default to false when the gutenburg support is released
+	);
+	register_post_meta( 'post', 'wikipediapreview_detectlinks', $options );
```

Comment:
once we have this option and default to true, the previous settings for `wikipediapreview_options_detect_links` is not necessary then? right?

Reply:
Right

Reply:
I have removed the code that adds the option since it won't be used but I kept the code that deletes the option for cleanup purposes.